src/Uno.UI/UI/Xaml/ObservableVectorWrapper.cs

```diff
@@ -14,7 +14,7 @@ namespace Windows.UI.Xaml
 #elif __ANDROID__
 	[Android.Runtime.Preserve(AllMembers = true)]
 #endif
-	internal abstract class ObservableVectorWrapper
+	internal class ObservableVectorWrapper
```

Comment:
You don't seem to use use it anywhere and it's  an `internal` class, is there a good reason for this change ?

Reply:
See above :)